static/bidder-params/resetdigital.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Relevant Digital Adapter Params",
+    "description": "A schema which validates params accepted by the Relevant Digital adapter",
+    "type": "object",
+    "properties": {}
```

Comment:
Based on adapter code, it seems like `placement_id` is the required bidder param. Should update `static/bidder-params/resetdigital.json` to include `placement_id` property

Refer following as example
https://github.com/prebid/prebid-server/blob/225e31e9afea25f8b844adcd3d274e8db3f7135e/static/bidder-params/appnexus.json#L1-L134